docs/curl/provenance.md

```diff
@@ -0,0 +1,829 @@
+    },
+    "KVs": [
+      {
+        "key": "key1",
+        "value": "eXl5",
+        "metadata": {
+          "version": {
+            "block_num": 4
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true,
+              "bob": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "key": "key1",
+        "value": "dGhpcyBpcyBhIGZpcnN0IHZhbHVl",
+        "metadata": {
+          "version": {
+            "block_num": 5
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "key": "key2",
+        "value": "dGhpcyBpcyBhIHNlY29uZCB2YWx1ZQ==",
+        "metadata": {
+          "version": {
+            "block_num": 5
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "key": "key3",
+        "value": "dGhpcyBpcyBhIHRoaXJkIHZhbHVl",
+        "metadata": {
+          "version": {
+            "block_num": 5
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "key": "key2",
+        "value": "dGhpcyBpcyBhIHNlY29uZCB2YWx1ZSB1cGRhdGVk",
+        "metadata": {
+          "version": {
+            "block_num": 6
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      },
+      {
+        "key": "key1",
+        "value": "dGhpcyBpcyBhIGZpcnN0IHZhbHVlIHVwZGF0ZWQ=",
+        "metadata": {
+          "version": {
+            "block_num": 6
+          },
+          "access_control": {
+            "read_users": {
+              "alice": true
+            },
+            "read_write_users": {
+              "alice": true
+            }
+          }
+        }
+      }
+    ]
+  },
+  "signature": "MEUCIQDaMpeJhgVlFOS0AL7AOd2wviQ6c9R/PrOH51grBifRlgIgL4hLscpLE+JNAryd0it+WkuFmiQyUHbg25/Z+T9ty3A="
+}
+```
+
+As you can see, user `alice` wrote 3 times to key `key1`, in blocks 4, 5 and 6, two times to `key2`, in blocks 5 and 6 and one time to `key3`, in block 5
+
+### Key access queries
+Another important view on data history is to know what user accessed specific key over time. Two next queries provide us with this information.
+
+### Query for key readers
+To query for users who ever read specific key, we use `/provenance/data/readers/{dbname}/{key}` GET query.
```

Comment:
To query for all the users that ever read a specific key, we use `/provenance/data/readers/{dbname}/{key}` GET query. 
This will return ... short bla bla

Reply:
done